nupic/frameworks/opf/clamodel.py

```diff
@@ -419,7 +423,7 @@ def run(self, inputRecord):
     if self.__logger.isEnabledFor(logging.DEBUG):
       self.__logger.debug("inputRecord: %r, results: %r" % (inputRecord,
                                                             results))
-
+    results.iterationTime = time.clock() - startTime
```

Comment:
@breznak - That would certainly limit the impact on the code. For the normal use case it would be accurate enough (it just needs to be accurate relative to other models being swarmed) but doesn't seem like a perfect solution either.